### ISSUE TITLE
Ignore checks per workspace

### ISSUE THREAD

**Issue by davinerd** [aquasecurity/tfsec#907]
**Is your feature request related to a problem? Please describe.**
There is no way to ignore a check for a specific workspace.

**Describe the solution you'd like**
Same as the `--tfvars-file`, I'd like to be able to ignore a test in an environment but not on another maybe via flags or by taking into account the current workspace.

**Describe alternatives you've considered**
No alternatives have been considered.

**Additional context**
It's common to use workspaces to manage different environments. One may want to ignore a test (i.e.`GCP009`) in staging but not in production.

Right now, by adding the (i.e.) `#tfsec:ignore:GCP009` to the tf code it'll be applied to every workspace, creating a situation where you are forced to have a staging/production parity environment, which not everyone wants to.

**Comment by owenrumney** [comment]
Hey @davinerd  - you can use the --config-file <path_to_config_file> for the environment runs then in the config file do

```yaml
---
exclude:
  - GCP009
```
 See https://tfsec.dev/docs/config/ for more

**Comment by davinerd** [comment]
@owenrumney thanks for the quick reply.

To my understanding, that would exclude the check for all resources, right?

**Comment by owenrumney** [comment]
@davinerd - that's correct, there isn't a mechanism for specifying ignores in a config targetted to specific resources.

We could do something like; 

```hcl
#tfsec:ignore:AWS018:ws:staging                  
resource "aws_security_group" "no_description" {
  # resource with no description
}
```

then if you run tfsec with `--workspace` we could go for

```shell
tfsec --workspace=staging .
```

Do you think that would tick your box?

**Comment by davinerd** [comment]
That would be perfect, yes. And easy to integrate into a CD/CI pipeline as well.

**Comment by owenrumney** [comment]
@davinerd  - this is now available in `v0.50.4`

**Comment by davinerd** [comment]
I'm amazed by how fast this has been proposed and merged into main: it's a pleasure to see such commitment to growing the tool this fast.

Thank you!
